src/main/scala/ai/privado/passes/HTMLParserPass.scala

```diff
@@ -85,6 +86,12 @@ class HTMLParserPass(cpg: Cpg, projectRoot: String, ruleCache: RuleCache) extend
       val webClient = new WebClient(
         new BrowserVersion.BrowserVersionBuilder(BrowserVersion.CHROME).setOnLine(false).build()
       )
+      val options = webClient.getOptions
+
+      // Configure options to ignore certain errors
+      options.setThrowExceptionOnFailingStatusCode(false) // Do not throw exceptions for failing HTTP status codes
+      options.setThrowExceptionOnScriptError(false)       // Do not throw exceptions for JavaScript errors
```

Comment:
Does this setting help in minimizing the HTML related errors we see when parsing code?

Reply:
Without this if any HTML file we getting errors we loosing all the HTML Elements inside that file.
But with this setting even though the errors come but we will able to get those HTML Elements inside it.

Not going to remove any errors we are showing in console.

Reply:
https://github.com/Privado-Inc/privado-core/pull/742/files#diff-b5040cb043e63e7c73b2468b4d9c3891539c8b4a22f18dc66166c58920b2348eL93

We can remove console logs if we change this one.

Reply:
log.debug is fine